Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -111,11 +112,15 @@
     field,
     trackSelectorConfig=(
         # fmt: off
-        TrackSelectorConfig(absEta=(None, 2.0), pt=(0.9 * u.GeV, None), nMeasurementsMin=9, maxHoles=2, maxSharedHits=2),
-        TrackSelectorConfig(absEta=(None, 2.6), pt=(0.4 * u.GeV, None), nMeasurementsMin=8, maxHoles=2, maxSharedHits=2),
-        TrackSelectorConfig(absEta=(None, 4.0), pt=(0.4 * u.GeV, None), nMeasurementsMin=7, maxHoles=2, maxSharedHits=2),
+        TrackSelectorConfig(absEta=(None, 2.0), pt=(0.9 * u.GeV, None), nMeasurementsMin=9, maxHoles=2, maxOutliers=2, maxSharedHits=2),
+        TrackSelectorConfig(absEta=(None, 2.6), pt=(0.4 * u.GeV, None), nMeasurementsMin=8, maxHoles=2, maxOutliers=2, maxSharedHits=2),
+        TrackSelectorConfig(absEta=(None, 4.0), pt=(0.4 * u.GeV, None), nMeasurementsMin=7, maxHoles=2, maxOutliers=2, maxSharedHits=2),
         # fmt: on
     ),
+    ckfConfig=CkfConfig(
+        seedDeduplication=True,
+        stayOnSeed=True,
+    ),
```

Comment:
If this argument is moved before `trackSelectorConfig`, can be simplified slightly:
```suggestion
    CkfConfig(
        seedDeduplication=True,
        stayOnSeed=True,
    ),
```